### ISSUE TITLE
Installing on CentOS 8 fails

### ISSUE THREAD

**Issue by indigo423** [opennms-forge/opennms-install#21]
When installing on a minimal CentOS 8 server the following error message is shown:

```
Error:
 Problem: cannot install the best candidate for the job
  - nothing provides libpng15.so.15(PNG15_0)(64bit) needed by rrdtool-1.7.1-0.1.el7.x86_64
  - nothing provides libpng15.so.15()(64bit) needed by rrdtool-1.7.1-0.1.el7.x86_64
  - nothing provides libdbi.so.0()(64bit) needed by rrdtool-1.7.1-0.1.el7.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
FAILED - Please check the bootstrap.log file for detailed error messages.
```

**Comment by indigo423** [comment]
Resolved with #22

**Comment by indigo423** [comment]
Resolved with #22
